website/blog/2020-07-06-version-0.63.md

```diff
@@ -0,0 +1,153 @@
+  This is a label
+</Text>
+```
+
+<p style="text-align: center; font-size: 12px">
+Sets the color of the Text component to labelColor as defined by iOS.
+</p>
+
+Android, on the other hand, [provides colors like colorButtonNormal](https://developer.android.com/reference/android/R.attr#colorButtonNormal). You can use this color in React Native with:
+
+```jsx
+import { View, Text, PlatformColor } from 'react-native';
+
+<View
+  style={{
+    backgroundColor: PlatformColor('?attr/colorButtonNormal')
+  }}>
+  <Text>This is colored like a button!</Text>
+</View>;
+```
+
+<p style="text-align: center; font-size: 12px">
+Sets the background color of the View component to colorButtonNormal as defined by Android.
+</p>
+
+You can learn more about `PlatformColor` from [the documentation](https://reactnative.dev/docs/next/platformcolor). You can also check the actual [code examples present in the RNTester](https://github.com/facebook/react-native/blob/master/RNTester/js/examples/PlatformColor/PlatformColorExample.js).
+
+`DynamicColorIOS` is an iOS only API that lets you define which color to use in light and dark mode. Similar to `PlatformColor`, this can be used anywhere you can use colors. `DynamicColorIOS` uses iOS’s `colorWithDynamicProvider` under the hood.
+
+```jsx
+import { Text, DynamicColorIOS } from 'react-native';
+
+const customDynamicTextColor = DynamicColorIOS({
+  dark: 'lightskyblue',
+  light: 'midnightblue'
+});
+
+<Text style={{ color: customDynamicTextColor }}>
+  This color changes automatically based on the system theme!
+</Text>;
+```
+
+<p style="text-align: center; font-size: 12px">
+Changes the text color based on the system theme
+</p>
+
+You can learn more about `DynamicColorIOS` from [the documentation](https://reactnative.dev/docs/next/dynamiccolorios).
+
+## Dropping iOS 9 and Node.js 8 support
+
+After over four years from its release, we are dropping support for iOS 9. This change will allow us to move faster by being able to reduce the number of compatibility checks that need to be placed in the native code to detect whether a given feature was supported on a certain iOS version. With its [market share of 1%](https://david-smith.org/iosversionstats/), it shouldn’t have much negative impact on your customers.
+
+At the same time, we are dropping support for Node 8. [Its LTS maintenance cycle expired in December 2019](https://nodejs.org/fr/blog/release/v8.9.0/). The current LTS is Node 10 and it is now the version that we are targeting. If you are still using Node 8 for the development of React Native applications, we encourage you to upgrade in order to receive all the latest security fixes and updates.
+
+## Other notable improvements
+
+- **Support rendering <View /> in <Text /> without explicit size**: You can now render any <View /> inside any <Text /> component without setting its width and height explicitly, which wasn’t always possible. On previous releases of React Native, this would result in a RedBox.
```

Comment:
<View /> and <Text /> don't show up.  Perhaps it needs to be in ``?
![image](https://user-images.githubusercontent.com/13405567/86595651-0a166d00-bf67-11ea-9e68-4a1f34fd3c6f.png)

Reply:
Good catch!

Reply:
I got warning while building about missing components and didn't really connect the dots!